macOS/geotoad.rb

```diff
@@ -0,0 +1,21 @@
+class Geotoad < Formula
+  desc "Query tool to query the geocaching.com website"
+  homepage "https://github.com/steve8x8/geotoad"
+  url "https://github.com/steve8x8/geotoad/archive/3.25.4.tar.gz"
```

Comment:
Do I smell a chicken-and-egg problem here? Tagging a whole tree will produce an archive which has to be checksummed, and the checksum to be merged into a file that has been tagged. As a result, the tag won't be consistent anymore. Re-tag? Slim chance...